Maquetado/scss/application.css

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
hmmmm something wen't wrong and this shouldn't be added to github. check the gitignore and delete this file.